crates/libs/windows/src/Windows/Win32/Data/HtmlHelp/mod.rs

```diff
@@ -1,18 +1,46 @@
 #![allow(non_snake_case, non_camel_case_types, non_upper_case_globals, clashing_extern_declarations, clippy::all)]
 pub const CLSID_IITCmdInt: ::windows::core::GUID = ::windows::core::GUID::from_u128(0x4662daa2_d393_11d0_9a56_00c04fb68bf7);
+#[cfg(feature = "guid_hashmap")]
+inventory::submit! { crate :: core :: GuidConst { name : stringify ! ( CLSID_IITCmdInt ) , guid : :: windows :: core :: GUID::from_u128(0x4662daa2_d393_11d0_9a56_00c04fb68bf7) , } }
```

Comment:
`rustfmt` seems to be avoiding formatting macro input, which is unfortunate here. I could format it before it gets passed to `quote` to fix this if that'd be preferred.